contracts/rewards/SubgraphAvailabilityManager.sol

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.7.6;
```

Comment:
Yes, I tried that but got an error because of that:

```
contracts/rewards/SubgraphAvailabilityManager.sol (^0.8.16) imports 
contracts/governance/Governed.sol (^0.7.6)  and contracts/rewards/IRewardsManager.sol (^0.7.6)
```

Reply:
Yeah let's stick to 0.7.6 for now in this repo - using different versions has proven problematic.